src/stories/Library/material-grid/MaterialGrid.tsx

```diff
@@ -0,0 +1,50 @@
+import clsx from "clsx";
+import React, { useEffect } from "react";
+import {
+  RecommendedMaterial,
+  RecommendedMaterialProps,
+} from "../recommended-material/RecommendedMaterial";
+
+export type MaterialGridProps = {
+  title: string;
+  amountOfMaterialsToShow: 4 | 8 | 12 | 16 | 20 | 24 | 28 | 32;
+  materials: RecommendedMaterialProps[];
+};
+
+export const MaterialGrid: React.FC<MaterialGridProps> = ({
+  title,
+  amountOfMaterialsToShow,
+  materials,
+}) => {
+  useEffect(() => {
+    require("./init-material-grid");
+  }, []);
+
+  const showMoreButton = materials.length > 8;
+
+  return (
+    <div className={clsx("material-grid")}>
```

Comment:
Should this be a section for semantic reasons? and then I see no reason to use clsc when there is only one class?

Reply:
We add a section around it with the paragraph rendering in drupal. But yes 👍